src/stories/Library/Lists/list-find-on-shelf/list-find-on-shelf.scss

```diff
@@ -0,0 +1,87 @@
+.find-on-shelf {
+  background-color: $c-global-secondary;
+
+  &__header-row {
+    display: none;
+
+    @include breakpoint-s {
+      border-bottom: solid 1px $c-global-primary;
+      display: flex;
+      height: $s-xl;
+      justify-content: flex-start;
+      align-items: center;
+    }
+
+    & > span {
+      &:first-of-type {
+        margin-left: $s-lg;
+        width: 404px;
+
+        @include breakpoint-s {
+          margin-left: $s-xl;
+        }
+      }
+      &:last-of-type {
+        margin-right: $s-md;
+        margin-left: auto;
+      }
+    }
+  }
+
+  &__row {
+    min-width: $s-6xl;
+    display: flex;
+    flex-direction: column;
+    border-bottom: solid 1px $c-global-primary;
+
+    @include breakpoint-s {
+      flex-direction: row;
+      min-height: 110px;
+      align-items: center;
+      justify-content: flex-start;
+    }
+
+    &:last-of-type {
+      border-bottom: none;
+    }
+
+    & > span {
+      margin: $s-md $s-md 0 $s-md;
+
+      &:first-of-type {
+        margin-top: $s-lg;
+      }
+
+      &:last-of-type {
+        margin-bottom: $s-lg;
+      }
+
+      @include breakpoint-s {
+        margin: 0;
+        max-width: 600px;
+        text-overflow: ellipsis;
+        overflow-y: hidden;
+
+        &:first-of-type {
+          margin-left: $s-xl;
+          margin-top: inherit;
+          width: 404px;
+          min-width: 404px;
+        }
+
+        &:last-of-type {
+          margin-bottom: inherit;
+          margin-left: auto;
+          margin-right: $s-xl;
+          overflow-y: visible;
+        }
+      }
+
+      & .hide-on-desktop {
+        @include breakpoint-s {
+          display: none;
+        }
+      }
```

Comment:
Please restructure this.

It is not very BEM like to combine `.find-on-shelf` with another class outside of the block. I suggest making `.hide-on-desktop` a utility class like `.hide-visually` in `shared.css`. An option might be to add a modifer to `.hide-visually` which only applies on desktop.

Reply:
Great point! It is a utility class now :)